content/en/docs/reference/access-authn-authz/abac.md

```diff
@@ -130,22 +130,21 @@ up the verbosity:
 
 ## A quick note on service accounts
 
-A service account automatically generates a user. The user's name is generated
-according to the naming convention:
+Creating a service account leads to generation of a new user.
+
+The user's name is generated according to the naming convention:
 
 ```shell
 system:serviceaccount:<namespace>:<serviceaccountname>
 ```
-Creating a new namespace also causes a new service account to be created, of
-this form:
+Creating a new namespace leads to creation of a new service account, which is of this form:
```

Comment:
Perhaps say: Creating a new namespace leads to the creation of a new service account in the following format: